integration-tests/smoke/ccip_test.go

```diff
@@ -106,3 +105,150 @@ func TestInitialDeployOnLocal(t *testing.T) {
 
 	// TODO: Apply the proposal.
 }
+
+func TestTokenTransfer(t *testing.T) {
+	lggr := logger.TestLogger(t)
+	ctx := ccdeploy.Context(t)
+	tenv, _, _ := testsetups.NewLocalDevEnvironment(t, lggr)
+
+	e := tenv.Env
+	state, err := ccdeploy.LoadOnchainState(e)
+	require.NoError(t, err)
+
+	feeds := state.Chains[tenv.FeedChainSel].USDFeeds
+	tokenConfig := ccdeploy.NewTokenConfig()
+	tokenConfig.UpsertTokenInfo(ccdeploy.LinkSymbol,
+		pluginconfig.TokenInfo{
+			AggregatorAddress: cciptypes.UnknownEncodedAddress(feeds[ccdeploy.LinkSymbol].Address().String()),
+			Decimals:          ccdeploy.LinkDecimals,
+			DeviationPPB:      cciptypes.NewBigIntFromInt64(1e9),
+		},
+	)
```

Comment:
Do we need that Weth token info from the other PR? https://github.com/smartcontractkit/chainlink/pull/15131